README.md

```diff
@@ -175,6 +175,13 @@ npm run docs
 
 The documentation uses [Jaguar.js](https://github.com/pixijs/jaguarjs-jsdoc) and the jsdoc format, the configuration file can be found at [scripts/jsdoc.conf.json](scripts/jsdoc.conf.json)
 
+### How to set up for development ###
+
+As of v5, PixiJS uses [Lerna](https://github.com/lerna/lerna) for development.
+To get started, install Lerna globally (`npm install -g lerna`), and run
+`lerna bootstrap`. This will install all the dependencies for all packages in
+this repository.
```

Comment:
I would like to remove this note about installing Lerna globally. Lerna is a devDependency and should work with `npm install` (this runs bootstrapping). Other than that we don't use Lerna that much.

Reply:
:+1: will remove and squash